.circleci/config.yml

```diff
@@ -28,8 +28,9 @@ commands:
       - run:
           name: Core Rust Checks
           command: |
-            cargo audit
             cargo fmt -- --check
+            echo "Skip audit for now since hawk 3.1.0 -> ring 0.16.11 -> abandoned `spin`"
```

Comment:
The spin +1 other audit on this branch are just warnings, so we can keep audit on

Reply:
Ok, was getting a fatal off of that, but will retry.

Reply:
I recall a fatal on this branch at one point but I think it was something else now gone